sonar-xml-plugin/src/main/java/org/sonar/plugins/xml/checks/LineLengthCheck.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * SonarQube XML Plugin
+ * Copyright (C) 2010-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.plugins.xml.checks;
+
+import java.util.Collections;
+import org.sonar.check.Rule;
+import org.sonar.check.RuleProperty;
+import org.sonar.plugins.xml.Utils;
+import org.sonarsource.analyzer.commons.xml.XmlFile;
+import org.sonarsource.analyzer.commons.xml.XmlTextRange;
+import org.sonarsource.analyzer.commons.xml.checks.SonarXmlCheck;
+
+/**
+ * RSPEC-103
+ */
+@Rule(key = LineLengthCheck.RULE_KEY)
+public class LineLengthCheck extends SonarXmlCheck {
+
+  public static final String RULE_KEY = "S103";
+
+  @RuleProperty(
+    key = "S103",
+    description = "The maximum authorized line length",
+    defaultValue = "120",
+    type = "INTEGER")
+  private int maximumLineLength;
```

Comment:
you need to initialize the variable with a default value, in case the rule has not been parameterized.
If a parameter is set in SonarQube, then the setMax... method will be called.